### ISSUE TITLE
Power consumption nRF52

### ISSUE THREAD

**Issue by StevenGhe** [zephyrproject-rtos/zephyr#35360]
Hello there,

Past few days/week I've been looking into how to limit power consumption using my Adafruit Feather Express nRF52840. I've asked in the general Slack but no response.

Currently I have about 1.1 mA current idle, and I'd like to 'sleep' the device as much as possible.
Looking at the Power Management section from Zephyr I understood you can either choose to set a power state to the system or to certain devices from your system.

Using `pm_power_state_set()` I'm able to set the power state. However after testing and looking into the code, nRF52 only supports `SYS_POWER_STATE_DEEP_SLEEP_1` which results in powering the system completely off (relevant [power.c](https://github.com/nrfconnect/sdk-zephyr/blob/master/soc/arm/nordic_nrf/nrf52/power.c) file). This is too much, as I'd like to keep the memory/context the same.

Is there another way of putting this device to sleep besides pm_power_state_set? As I'm using the bluetooth stack, as far as I am aware there is no Bluetooth device in my build. There is no way to way to disable bluetooth ([related issue on disabling Bluetooth](https://github.com/zephyrproject-rtos/zephyr/issues/3192)), so I'm kind of stuck, unable to limit power consumption.

Sorry if this 'blank' issue is related to a certain topic, I was not sure which one it belonged to.

Thanks in advance,

Steven

**Comment by gmarull** [comment]
Have you checked https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/boards/nrf/system_off ?

**Comment by ceolin** [comment]
Based on first comment in https://github.com/zephyrproject-rtos/zephyr/issues/31162 this seems to be the only runtime state supported by Nordic boards. @gmarull do you know if this target supports other states that are just not implemented ?

**Comment by ceolin** [comment]
@StevenGhe  btw, you are probably using an old Zephyr version. The way pm states are implemented now is very different.

**Comment by StevenGhe** [comment]
As soon as I looked at the sample gmarull showed, I realized that the newer versions indeed have a lot of changes.

Quite sure I will be able to do some sleep/idle current reduction with the presented sample, if any issues arise I will make another ticket :).

@ceolin, the reason I created this ticked that other power.c files like for example [this stm32l4 one](https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/st_stm32/stm32l4/power.c) had way more features. And given the nRF52 popularity it seemed weird it had next to nothing.
